File: smelly/ignored-test/CSharpXUnit/CSharpXUnit.Tests/Tests/LogAfterAwait_ShouldCleanMethodNameAsync5.cs
using Xunit;

namespace DefaultNamespace
{
    public class CallSiteTests
    {
        [Fact(Skip = "")]
        public void LogAfterAwait_ShouldCleanMethodNameAsync5()
        {
            {
                //namespace en name of current method
                const string currentMethodFullName = "NLog.UnitTests.LayoutRenderers.CallSiteTests.AsyncMethod5";
                var logFactory = new LogFactory().Setup().LoadConfigurationFromXml(@"
                       <nlog>
                           <targets><target name='debug' type='Debug' layout='${callsite}|${message}' /></targets>
                           <rules>
                               <logger name='*' levels='Debug' writeTo='debug' />
                           </rules>
                       </nlog>").LogFactory;
                AsyncMethod5(logFactory).Wait();
                logFactory.AssertDebugLastMessage($"{currentMethodFullName}|dude");
                new InnerClassAsyncMethod5().AsyncMethod5(logFactory).Wait();
                logFactory.AssertDebugLastMessage($"{typeof(InnerClassAsyncMethod5).ToString()}.AsyncMethod5|dude");
            }
        }
    }
}

File: refactored/ignored-test/python-pytest/LogAfterAwait_ShouldCleanMethodNameAsync5.py
import pytest

class TestCallSite:

    # @pytest.mark.skip(reason="")
    def test_log_after_await_should_clean_method_name_async5(self):
        current_method_full_name = "NLog.UnitTests.LayoutRenderers.CallSiteTests.AsyncMethod5"
        log_factory = LogFactory().setup().load_configuration_from_xml("<nlog>\n" +
            "    <targets><target name='debug' type='Debug' layout='${callsite}|${message}' /></targets>\n" +
            "    <rules>\n" +
            "        <logger name='*' levels='Debug' writeTo='debug' />\n" +
            "    </rules>\n" +
            "</nlog>").get_log_factory()
        AsyncMethod5(log_factory).get()
        log_factory.assert_debug_last_message(current_method_full_name + "|dude")
        InnerClassAsyncMethod5().async_method5(log_factory).get()
        log_factory.assert_debug_last_message(f"{InnerClassAsyncMethod5.__class__}|dude")
